packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -493,8 +509,7 @@ export class Viewer {
       this.calculateComponentDimensions();
       this.isResizing = false;
 
-      // TODO(dan): Need to add and invoke message for resizing the image
-      // stream. https://vertexvis.atlassian.net/browse/SDK-921
+      this.stream.updateDimensions({ dimensions: this.dimensions });
```

Comment:
Will this throw errors if the WS connection hasn’t been established?

Reply:
probably so. is there now a way to check if the connection has been established and 'connected'?

Reply:
would probably be nice to have a helper function in the `Viewer` component or `StreamApi`, but you can check the existence of `streamDisposable`.

Reply:
updated in fcddbcf289ccdc6276ad59d7afa8c0ec7521b105